sse_solver_py/src/lib.rs

```diff
@@ -203,12 +209,62 @@ fn solve_sse_bra_ket(
     Ok(config.simulate_system(&initial_state, &system))
 }
 
+#[pyfunction]
+#[allow(clippy::too_many_arguments)]
```

Comment:
remove this if we are adding it as a method in solve_sse. Otherwise just remove it from the enum on line 75